### DISCUSSION TITLE
Timeline Bucket API Returns Null

### DISCUSSION THREAD

**Discussion by tbelway** [immich-app/immich#14149] [Q&A]
### The bug

If you run:
curl -s -H "x-api-key: ${REDACTED_API_KEY}" -H "accept: application/json" "${REDACTED_URL}/api/timeline/bucket?size=MONTH&timeBucket=${DATE_YYYY-MM-DD}"

Then a null value is returned instead of the expected geodata.

### The OS that Immich Server is running on

Fedora Server

### Version of Immich Server

v1.120.1

### Version of Immich Mobile App

N/A

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
Using quadlets.
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

1. Create API key.
2. curl the timeline api with defined buckets and with api key.
3. Get null value returned.

### Additional information

Related dawarich issue:
https://github.com/Freika/dawarich/issues/270

**Comment by jrasm91** [comment]
The timeline endpoint is working as designed and you are probably expecting to see an asset that is being omitted from the query.

**Reply by tbelway** [reply]
Hey there,

If I use any dates at all I get null returned. Can you expand on your statement? I have no had any values returned since... I want to say v1.118...?

:edit:

I see I'm out of date, I see you commented on teh dawarich thread... let me get it tested first...

**Reply by tbelway** [reply]
Freika did a rework and instead of pulling from buckets it pulls from metadata and it works now thanks so much :)
